lib/recog/verifier.rb

```diff
@@ -15,19 +15,19 @@ def verify
         fp.verify_params do |status, message|
           case status
           when :warn
-            reporter.warning "WARN: #{message}", fp.line
```

Comment:
I moved the WARN/FAIL tags for output into verify_reporter.rb instead, since that is what is actually logging the text anyway. 

Needed to do this so that we could share a formatter and reporter between the XML schema validator and recog's verifier (don't repeat yourself/DRY, yadda yadda).

Reply:
Nice work pulling the label out and simplifying the `reporter` calls!